index.js

```diff
@@ -13,6 +13,9 @@ export default function toCSS(obj) {
       value = `${value}px`;
     }
 
-    return `${key}:${value} !important;`;
+    const isTouch = ('ontouchstart' in window) || window.DocumentTouch && document instanceof DocumentTouch;
```

Comment:
Hey @RicheyRyan I've changed based on this https://github.com/Modernizr/Modernizr/blob/master/feature-detects/touchevents.js